ospfd/ospf_route.h

```diff
@@ -69,6 +69,8 @@ struct route_standard {
 
 	/*  */
 	uint8_t flags; /* From router-LSA */
+
+	bool transit;
```

Comment:
Do we need this field to be added? Isn't the fields path_type as OSPF_PATH_INTRA_AREA and type as OSPF_DESTINATION_NETWORK means it is a transit route?

Reply:
I'm sorry that I'm not so familiar with OSPF protocol. Fixed.